src/SprykerEco/Service/CrefoPay/CrefoPayConfig.php

```diff
@@ -25,4 +36,34 @@ public function getCrefoPayOrderIdLength(): int
     {
         return static::CREFO_PAY_ORDER_ID_LENGTH;
     }
+
+    /**
+     * @api
+     *
+     * @return int
+     */
+    public function getCrefoPayUserIdMaxLength(): int
+    {
+        return static::CREFO_PAY_USER_ID_MAX_LENGTH;
+    }
+
+    /**
+     * @api
+     *
+     * @return int
+     */
+    public function getCrefoPayBasketItemIdMaxLength(): int
+    {
+        return static::CREFO_PAY_BASKET_ITEM_ID_MAX_LENGTH;
+    }
+
+    /**
+     * @api
+     *
+     * @return bool
+     */
+    public function isBusinessToBusiness(): bool
```

Comment:
please check the name in the changelogs
Also for non unique class names FQCN must be used in changelogs

Reply:
Also would be nice to have specification for this one. It's not self explanatory

Reply:
@gechetspr updated.